src/constants.js

```diff
@@ -0,0 +1,4 @@
+export const ELEMENT_TYPE = {
```

Comment:
Instead of exporting a single constant - you can export separate constants as it's easy to import only what's needed - this makes is easier to treeshake too. 

 ```suggestion
export const FILE_JAVASCRIPT = 'text/javascript';
export const FILE_CSS = 'text/css';
```

Now you're only able to import what you need which won't be a problem even if you have large number of constants in the same file.